core/Symbols.cc

```diff
@@ -32,7 +32,7 @@ bool SymbolRef::operator!=(const SymbolRef &rhs) const {
 vector<TypePtr> Symbol::selfTypeArgs(const GlobalState &gs) const {
     ENFORCE(isClassOrModule()); // should be removed when we have generic methods
     vector<TypePtr> targs;
-    for (auto tm : typeMembers()) {
+    for (auto &tm : typeMembers()) {
```

Comment:
`SymbolRef` is cheap to copy

Reply:
```suggestion
    for (auto tm : typeMembers()) {
```

Reply:
Bah, somehow I think I determined that `typeMembers()` was returning a vector of `TypePtr`; Code led me astray!

Reply:
clangd is A+ here, it told me the type.